python/swigfaiss.swig

```diff
@@ -118,13 +121,16 @@ namespace std {
         size_t size();
         T at (size_t n) const;
         void resize (size_t n);
+        void swap (vector<T> & other);
```

Comment:
could this be the source of the CI error?

Reply:
Yes. C.f. https://github.com/swig/swig/issues/568